lib/bundle.js

```diff
@@ -173,8 +175,9 @@ function logTree(tree) {
   ui.log('info', '');
 }
 
-function extractBundleName(fileName) {
-  return path.relative(config.pjson.baseURL, fileName.replace(/\.js$/, '')).replace(/\\/g, '/');
+function extractBundleName(fileName, builder) {
+  var file = path.resolve(config.pjson.baseURL, fileName).replace(/\\/g, '/');
```

Comment:
Actually, you are right here. As `toFileURL` handles `\` replacement, there's nothing much left to `extractBundleName`. Besides, `exaractBundleName` requires `builder` as a parameter, which is somewhat ugly. So, inlining makes much more sense.

Making the changes.
`      var bundleName = systemBuilder.getCanonicalName(toFileURL(path.resolve(config.pjson.baseURL, fileName)));
`